packages/client/src/declarations/submissionMiddleware.test.ts

```diff
@@ -158,6 +158,7 @@ describe('Submission middleware', () => {
           submissionStatus: ACTION_STATUS_MAP[submissionAction]
         })
         await middleware(action)
+        await new Promise((resolve) => setTimeout(resolve, 2000))
```

Comment:
@rabiulislamanik Instead of waiting for 2 seconds, can we use this please? https://vitest.dev/guide/mocking.html#timers